migrations/versions/550715127385_.py

```diff
@@ -0,0 +1,50 @@
+"""add 'contact_information' table and flesh out 'suppliers'
+
+Revision ID: 550715127385
+Revises: 3d5aabf7d291
+Create Date: 2015-04-13 17:42:52.688791
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '550715127385'
+down_revision = '3d5aabf7d291'
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+def upgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('contact_information',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('supplier_id', sa.Integer(), nullable=True),
+    sa.Column('contact_name', sa.String(), nullable=False),
+    sa.Column('phone_number', sa.String(), nullable=True),
+    sa.Column('email', sa.String(), nullable=True),
+    sa.Column('website', sa.String(), nullable=True),
+    sa.Column('address1', sa.String(), nullable=True),
+    sa.Column('address2', sa.String(), nullable=True),
+    sa.Column('city', sa.String(), nullable=True),
+    sa.Column('country', sa.String(), nullable=True),
+    sa.Column('postcode', sa.String(), nullable=True),
+    sa.ForeignKeyConstraint(['supplier_id'], ['suppliers.supplier_id'], ),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.add_column(u'suppliers', sa.Column('clients', postgresql.JSON(), nullable=True))
+    op.add_column(u'suppliers', sa.Column('duns_number', sa.String(), nullable=True))
+    op.add_column(u'suppliers', sa.Column('esourcing_id', sa.String(), nullable=True))
+    op.create_unique_constraint(None, 'suppliers', ['duns_number'])
+    op.create_unique_constraint(None, 'suppliers', ['esourcing_id'])
+    ### end Alembic commands ###
+
+
+def downgrade():
+    ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint(None, 'suppliers', type_='unique')
```

Comment:
Yep, looking at the scripts again I think it would definitely be best to wrap the three migrations here into a single migration file.